packages/server/src/index.ts

```diff
@@ -3,6 +3,9 @@ import { Api } from './api'
 import { App } from './app'
 import { Launcher } from './launcher'
 
+// Set NODE_ENV to production when starting messaging using the binary version
+process.env.NODE_ENV = (<any>process).pkg ? 'production' : process.env.NODE_ENV
```

Comment:
process.pkg is only defined when the server has been bundled by the pkg library!